src/main/groovy/de/tracetronic/jenkins/plugins/ecutestexecution/steps/StartToolStep.groovy

```diff
@@ -226,11 +203,11 @@ class StartToolStep extends Step {
                 connectTool(toolName)
```

Comment:
@tt-marcel 

startTool should be the only step that set the build status after throwing an error message. An test execution if no tool could be started or was found makes no sense at all